src/management/Akka.Management/Resources/reference.conf

```diff
@@ -9,12 +9,13 @@ akka.management {
   http {
     # The hostname where the HTTP Server for Http Cluster Management will be started.
     # This defines the interface to use.
-    # InetAddress.getLocalHost.getHostAddress is used not overriden or empty
+    # akka.remote.dot-netty.tcp.public-hostname is used if not overriden or empty.
+    # if akka.remote.dot-netty.tcp.public-hostname is empty, Dns.GetHostName is used.
```

Comment:
Ah, so the `http.hostname` here would be advertised as the "public" hostname for connecting to this service?

Reply:
There's `http.hostname` and `http.bind-hostname`, the effective bind host name is `http.bind-hostname`, if it is empty then it falls back to `http.hostname`.

Reply:
This `http.hostname` has to match exactly to the host names resolved by discovery because it is used to determine if a resolved host name is "self" and used as a marker and check to determine if a node should join itself to start a cluster

Reply:
Ahhh got it - yes, you can't ever use `0.0.0.0` or the IPV6 equivalent for a public hostname. Will never work. Makes sense.